rest/config_database_test.go

```diff
@@ -82,3 +83,48 @@ func TestDbConfigUpdatedAtField(t *testing.T) {
 	assert.Equal(t, registry.CreatedAt.UnixNano(), registryCreated.UnixNano())
 	assert.Greater(t, registry.UpdatedAt.UnixNano(), registryUpdated.UnixNano())
 }
+
+func TestConfigToBucketPointName(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("Need cbs bucket for this test")
+	}
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyHTTPResp, base.KeyHTTP)
+
+	rt := NewRestTester(t, nil)
+	defer rt.Close()
+	const testBucketName = "sg_int_my.Bucket"
```

Comment:
Can you make a function like this public in base/util_testing.go so we can use it like we do in a bucket pool?

```
	// Append a timestamp to all of the bucket names to ensure uniqueness across a single package.
	// Not strictly required, but can help to prevent (index) resources from being incorrectly reused on the server side for recently deleted buckets.
	bucketNameTimestamp := time.Now().UnixNano()

	// create required number of buckets (skipping any already existing ones)
	for i := 0; i < numBuckets; i++ {
		bucketName := fmt.Sprintf(tbpBucketNameFormat, tbpBucketNamePrefix, i, bucketNameTimestamp)
```

This is from main_test_bucket_pool.go ?		

I'm just trying to avoid a failure from a test collision.